### PR TITLE
Update the gradle properties to match the EAP release branch

### PR DESCRIPTION
**Pull request by jeremyjiang-dev**

### CODE REVIEW

File: firebase-perf/ktx/gradle.properties [hunk 1]
```diff
@@ -1,2 +1,3 @@
 android.enableUnitTestBinaryResources=true
-
+version=20.0.7-eap
```
Comment (rlazo) [inline firebase-perf/ktx/gradle.properties:2]: You can revert the changes to this file. It's only changed by the tool so there wouldn't be any conflict.

Comment (jeremyjiang-dev) [reply firebase-perf/ktx/gradle.properties]: Done!

**Review by rlazo — APPROVED** [review]
